### ISSUE TITLE
Create host fails on 2.5.2

### ISSUE THREAD

**Issue by jacac** [theforeman/foreman_fog_proxmox#207]
**Describe the bug**
Creating host cannot load nic interface form. 

**Versions:**
Foreman: 2.5.2
foreman_fog_proxmox: 0.14.0
fog-proxmox: 0.14.

**To Reproduce**
1. Go to 'Create Host'
2. Select "Deploy on" to a Proxmox cluster
3. Interface tab header turns red with the error `Error loading interfaces information: Internal Server Error`

**Backtrace and error attachached:**
[fog_proxmox_backtrace.txt](https://github.com/theforeman/foreman_fog_proxmox/files/6887391/fog_proxmox_backtrace.txt)

**Comment by RedChops** [comment]
I'm currently sitting on Foreman version 2.4. Does this issue mean I should not upgrade to 2.5 until you release version 0.15.0 of this plugin?

**Comment by fgoebel** [comment]
We started building new deployment infrastructure with foreman and also built a new Proxmox-Cluster to work with. We are using Foreman 3.1 and Proxmox 7.1 and I can also confirm that this Error on the Interfaces tab also happens for me.

I am not able to actually help with the actual changes at all but If I can help with for example testing early versions I am quite happy to do so. If there is something i can help here please contact me ;-)

Is there already some early versions to try? Thank you very much for your efford and work here!

**Comment by donywahlberg** [comment]
Found this thread through some web searching and wanted to chime in. 

Stood up a fresh ProxMox 7.1-7 cluster for homelabbing Foreman automation I use at work via RHEL Satellite. Using Foreman 3.1 with the most current versions of fog-promox and foreman-fog-proxmox I was able to begin spinning up VMs through Foreman that powered on as far as the boot menu. 

Upon further attempts to configure more finely for Kickstarting full server builds my Foreman GUI under "Create Host" began to exhibit this exact behavior. Tracing the bug in prod logs the only tangible information I can find to share here relates to the following error log ID returning as "Null". 

[W|app|fb1c8068]

I was able to fetch further info on the error using "foreman-rake errors:fetch_log request_id=fb1c8068" but have since blown up the Foreman VM to try moving back to older stable versions of each software. 

Commenting mostly to say as of 3/2022 I was able to generate working VMs using Foreman 3.1 and ProxMox 7.1-7, but for some reason which lines up with the information here the functionality broke itself while I was installing different plugins such as BootDisk. 

Upon wiping for a fresh server I can no longer generate VMs at all. Moving back to Foreman 2.4 and foreman-fog-proxmox v13.4 and will report results.

**Comment by donywahlberg** [comment]
Small update, I found a workaround after previously thinking this killed all functionality. If you attempt to provision the host without Interface settings it simply drops me back onto the Interface page sans error -- able to interact with the GUI and set IP related config. 

This at least confirms functionality for ProxMox 7.1-7 and Foreman 3.1 w/ foreman-fog_promox  and fog-proxmox versions .14 respectively. Good enough for my purposes, but I hope the context can provide some assistance to others working with this tech. 

Thanks for the contribution to the community.

**Comment by ahoiroman** [comment]
Hello,

is there any chance to see a new release for this soon?

**Comment by ahoiroman** [comment]
Hi @tristanrobert 

Is there anything we can do to get this to an updated release? How can we support you? Atm I am stuck with Foreman 2.4 due to this problem :-/

**Comment by Nascire** [comment]
@tristanrobert I had the same problem and manually applied the commit https://github.com/theforeman/foreman_fog_proxmox/commit/1c246c98016c9062308c98e0b9b9dfc98d1a3695 - provisioning is working fine, so if nothing speaks against, I would suggest to PR this one

**Comment by jacac** [comment]
After playing around with those forms it seems that timing could be causing this issue with `byte_size_f` and `counter_f` fields. Those fields are converted to React components and maybe not ready if they are inside fieldsets or nested fieldsets. I hope someone could shed some light to that.

**Comment by tristanrobert** [comment]
Yes react components have broken the feature: input forms could not be set. I have found a fix. I test it again and again before merging very soon.
